### PR TITLE
Add the privacy policy link

### PR DESCRIPTION
**Pull request by benoitchantre**
## Description

Addresses issue #210 by adding the privacy policy link in the footer.

privacy-policy

## List of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
- [x] This pull request relates to a ticket.
- [x] My code is tested.
- [x] I want my code added to WP Rig.

### CODE REVIEW

**Comment by felixarntz** [comment]
Good catch @benoitchantre!

@mor10 Any preference on how to include this? I personally think we should put it on the same line, simply separated by another `<span class="sep"> | </span>`.

**Comment by benoitchantre** [comment]
I'll update the PR if needed.

**Comment by benoitchantre** [comment]
@mor10 Ping

**Comment by benoitchantre** [comment]
@felixarntz I have update the PR to keep the Privacy Policy link on the same line. It's a the end to be more visible.

privacy-policy-link-at-the-end

**Comment by felixarntz** [comment]
@benoitchantre Can you double-check? It seems that the code is still the same as before. Did you forget to push? :)

**Comment by benoitchantre** [comment]
> Did you forget to push? :)

@felixarntz Correct.

File: footer.php [hunk 1]
```diff
@@ -23,8 +23,12 @@
...
 		<span class="sep"> | </span>
 		<?php
-			/* translators: 1: Theme name, 2: Theme author. */
-			printf( esc_html__( 'Theme: %1$s by %2$s.', 'wp-rig' ), '<a href="' . esc_url( 'https://github.com/wprig/wprig/' ) . '">WP Rig</a>', 'the contributors' );
+		/* translators: 1: Theme name, 2: Theme author. */
+		printf( esc_html__( 'Theme: %1$s by %2$s.', 'wp-rig' ), '<a href="' . esc_url( 'https://github.com/wprig/wprig/' ) . '">WP Rig</a>', 'the contributors' );
+
+		if ( function_exists( 'the_privacy_policy_link' ) ) :
+			the_privacy_policy_link( '<span class="sep"> | </span>', '' );
+		endif;
```
Comment (felixarntz) [inline footer.php]: Please use regular `if { }` syntax instead. The `if : endif;` is no longer preferred (also in WordPress core).

Comment (benoitchantre) [reply footer.php]: Thank you. That's changed.

**Pull request merged**